LinkServer/Controllers/AuthenticatorController.cs

```diff
@@ -31,6 +31,16 @@ public AuthenticatorController(AppLogger logger)
         // Static dictionary to store logged-in sessions (as an example)
         private static readonly ConcurrentDictionary<string, bool> _authenticatedUsers = new();
 
+        // Simulated packet reception method for demonstration
```

Comment:
The AuthenticatorController should only be dealing with authentication routes. A general POST route should be located in the UplinkController.cs file.

Reply:
copy that, I moved that POST method to the UplinkController.cs